CMakeLists.txt

```diff
@@ -1,5 +1,7 @@
 cmake_minimum_required(VERSION 2.8.0)
 
+set(CMAKE_CXX_STANDARD 14)
```

Comment:
Without this specified, gpxlib won't compile on Ubuntu 16.04 and spits out errors below:
```
$ make      
[  2%] Building CXX object gpx/CMakeFiles/gpx.dir/Node.cpp.o
In file included from gpxlib/gpx/Node.cpp:29:0:
gpxlib/gpx/../gpx/Node.h:175:40: error: ‘nullptr’ was not declared in this scope
     virtual Node *add(Report *report = nullptr);
                                        ^
gpxlib/gpx/../gpx/Node.h:186:69: error: ‘nullptr’ was not declared in this scope
     virtual Node *add(const char *name, Type type, Report *report = nullptr);
                                                                     ^
gpxlib/gpx/../gpx/Node.h:204:57: error: ‘nullptr’ was not declared in this scope
     virtual Node *insert(Node *before, Report *report = nullptr);
                                                         ^
gpxlib/gpx/../gpx/Node.h:216:86: error: ‘nullptr’ was not declared in this scope
     virtual Node *insert(Node *before, const char *name, Type type, Report *report = nullptr);
                                                                                      ^
gpxlib/gpx/../gpx/Node.h:226:54: error: ‘nullptr’ was not declared in this scope
     virtual void copy(Node *source, Report *report = nullptr);
                                                      ^
gpxlib/gpx/../gpx/Node.h:235:44: error: ‘nullptr’ was not declared in this scope
     virtual bool validate(Report *report = nullptr) const;
```